src/ess/reduce/nexus/types.py

```diff
@@ -192,10 +192,10 @@ class Filename(sciline.Scope[RunType, Path], Path): ...
 
 
 @dataclass
-class PulseSelection(Generic[RunType]):
+class TimeInterval(Generic[RunType]):
     """Range of neutron pulses to load from NXevent_data or NXdata groups."""
```

Comment:
For the logs in NXtransformations it is loading the full log. The time interval is later used to determine which values are relevant.

The workflow is not loading any other logs currently.

Reply:
Not really at the moment, since the label-based slicing in Scipp and ScippNeXus does not do what we need (include the previous value). We thus want to load "more" than the naive slice says. Unless we get very large logs it seems easier to just load everything, and then move events to log values.